<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>764</width>
    <height>542</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Boxee Simulator</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>764</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuBoxee_Simulator">
    <property name="title">
     <string>Boxee</string>
    </property>
    <addaction name="actionPower_On_Off"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuComm_Log">
    <property name="title">
     <string>Communication Log</string>
    </property>
    <property name="separatorsCollapsible">
     <bool>false</bool>
    </property>
    <addaction name="actionClear"/>
    <addaction name="actionSave_As"/>
   </widget>
   <widget class="QMenu" name="menuVirtual_Remote">
    <property name="title">
     <string>Virtual Remote</string>
    </property>
    <addaction name="actionPlay_Pause"/>
    <addaction name="separator"/>
    <addaction name="actionUp"/>
    <addaction name="actionRight"/>
    <addaction name="actionDown"/>
    <addaction name="actionLeft"/>
    <addaction name="separator"/>
    <addaction name="actionEnter"/>
    <addaction name="separator"/>
    <addaction name="actionMenu"/>
    <addaction name="separator"/>
    <addaction name="actionInput_Text"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_Boxee_Simulator"/>
   </widget>
   <addaction name="menuBoxee_Simulator"/>
   <addaction name="menuComm_Log"/>
   <addaction name="menuVirtual_Remote"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPower_On_Off"/>
   <addaction name="separator"/>
   <addaction name="actionPlay_Pause"/>
   <addaction name="separator"/>
   <addaction name="actionLeft"/>
   <addaction name="actionUp"/>
   <addaction name="actionDown"/>
   <addaction name="actionRight"/>
   <addaction name="actionEnter"/>
   <addaction name="separator"/>
   <addaction name="actionMenu"/>
   <addaction name="separator"/>
   <addaction name="actionInput_Text"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
  <action name="actionAbout_Boxee_Simulator">
   <property name="text">
    <string>About Boxee Simulator</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionPower_On_Off">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/power-on.png</normaloff>:/images/power-on.png</iconset>
   </property>
   <property name="text">
    <string>Power On/Off</string>
   </property>
   <property name="toolTip">
    <string>Power On/Off</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="menuRole">
    <enum>QAction::PreferencesRole</enum>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionClear">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/clear-log.png</normaloff>:/images/clear-log.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear Log</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/save-as.png</normaloff>:/images/save-as.png</iconset>
   </property>
   <property name="text">
    <string>Save As ...</string>
   </property>
   <property name="toolTip">
    <string>Save Log As...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionPlay_Pause">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/play.png</normaloff>:/images/play.png</iconset>
   </property>
   <property name="text">
    <string>Play / Pause</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionUp">
   <property name="icon">
    <iconset>
     <normalon>:/images/up.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRight">
   <property name="icon">
    <iconset>
     <normalon>:/images/right.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Right</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionDown">
   <property name="icon">
    <iconset>
     <normalon>:/images/down.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionLeft">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/left.png</normaloff>:/images/left.png</iconset>
   </property>
   <property name="text">
    <string>Left</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionInput_Text">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/input-text.png</normaloff>:/images/input-text.png</iconset>
   </property>
   <property name="text">
    <string>Input text ...</string>
   </property>
   <property name="toolTip">
    <string>Input text...</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionEnter">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/enter.png</normaloff>:/images/enter.png</iconset>
   </property>
   <property name="text">
    <string>Enter</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionMenu">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/menu.png</normaloff>:/images/menu.png</iconset>
   </property>
   <property name="text">
    <string>Menu</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
